Podcast/TopSearchResultsViewController.swift

```diff
@@ -68,6 +68,8 @@ class TopSearchResultsViewController: ViewController, UITableViewDataSource, UIT
         }
         
         searchResultsTableView.layoutIfNeeded()
+
+        mainScrollView = searchResultsTableView
```

Comment:
i'm getting large titles from search to episode detail (not sure if this is related) 

![image](https://user-images.githubusercontent.com/20246620/40214099-e5c9e8a2-5a26-11e8-88f7-618c5177364b.png)